js/main.js

```diff
@@ -0,0 +1,28 @@
+var $photoUrl = document.querySelector('.photo-url');
+var $img = document.querySelector('img');
+
+$photoUrl.addEventListener('input', function photoUrl(event) {
+  $img.setAttribute('src', $photoUrl.value);
+});
+
+
+var $form = document.querySelector('form');
+
+$form.addEventListener('submit', function journalEntry(event) {
+  event.preventDefault();
+
+  var nextEntry = {
+    title: $form.elements.title.value,
+    photoUrl: $form.elements.photo.value,
+    notes: $form.elements.notes.value,
+    nextEntryId: data.nextEntryId
```

Comment:
You should name your property `entryId`, not `nextEntryId`, as I discussed with the class yesterday